WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -532,6 +532,57 @@ - (WPMediaRequestID)imageWithSize:(CGSize)size completionHandler:(WPMediaImageBl
     return [self.mediaID intValue];
 }
 
+- (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler
+{
+    if (!completionHandler) {
+        return 0;
+    }
+
+    NSString *errorMessage;
+    // Check if asset being used is a video, if not this method fails
+    if (self.assetType != MediaTypeVideo) {
+        errorMessage = NSLocalizedString(@"Media selected is not a video.",@"Error message when user tries to preview an image media like a video");
+        completionHandler(nil, [self errorWithMessage:errorMessage]);
+        return 0;
+    }
+
+    NSURL *url = nil;
+    // Do we have a local url, or remote url to use for the video
+    if (self.absoluteLocalURL) {
+        url = [NSURL URLWithString:self.absoluteLocalURL];
+    } else if (self.remoteURL) {
+        url = [NSURL URLWithString:self.remoteURL];
+    }
+
+    if (!url) {
+        errorMessage = NSLocalizedString(@"Media selected is not available.",@"Error message when user tries a non longer existent video media object.");
```

Comment:
Suuuuper nitpicky – could we have an empty space after the comma in these localized strings?